### ISSUE TITLE
trainloader auto-padding the wav signal. Where is the underlying codes to modify padding logits? 

### ISSUE THREAD

**Issue by ZEngXCiao** [speechbrain/speechbrain#942]
![image](https://user-images.githubusercontent.com/38159923/130036423-575bf6a1-31b8-4619-8bdd-31bd5e18d648.png)

In the speaker_verification_plda.py, I do iteration in the set_loader. It will auto-complement the wav file to the same length. I want to know if the signal is two dimension, does it do this still correctly, and want to know where the auto-padding codes lying?

**Comment by ZEngXCiao** [comment]
![E3)RMEV{8}T3N(9 ~ LVJF](https://user-images.githubusercontent.com/38159923/130037092-e110517e-b2b1-4b1a-9131-14b632bd653d.png)

**Comment by ZEngXCiao** [comment]
The image shows codes in the function of emb_computation_loop() of speaker_verification_plda.py.  More Specifically, it's wavs, lens = batch.sig

**Comment by TParcollet** [comment]
Certainly a question for @popcornell or @Gastron  :-)

**Comment by csukuangfj** [comment]
> want to know where the auto-padding codes lying?

See

https://github.com/speechbrain/speechbrain/blob/6cbef262bdd6c597f8670b1fc59c51f31e3982de/speechbrain/dataio/batch.py#L106

https://github.com/speechbrain/speechbrain/blob/6cbef262bdd6c597f8670b1fc59c51f31e3982de/speechbrain/dataio/batch.py#L125

https://github.com/speechbrain/speechbrain/blob/6cbef262bdd6c597f8670b1fc59c51f31e3982de/speechbrain/utils/data_utils.py#L414

https://github.com/speechbrain/speechbrain/blob/6cbef262bdd6c597f8670b1fc59c51f31e3982de/speechbrain/utils/data_utils.py#L356

**Comment by ZEngXCiao** [comment]
Thank you much!
